src/js/components/atoms/tooltip/tooltip.unit.spec.ts

```diff
@@ -4,19 +4,35 @@ import { build } from 'vuenit';
 
 describe('components', () => {
     describe('atoms', () => {
-        describe('tooltip', () => {
-            const mount = build(Tooltip);
+        let mount;
+        let vm;
+
+        beforeEach(() => {
+            mount = build(Tooltip);
+        });
 
+        afterEach(() => {
+            vm.$destroy();
+        })
+
+        describe('tooltip', () => {
             it('should render a tooltip with a block property', () => {
-                const vm = mount({
```

Comment:
My advise would be not to reusing the viewModel between states, because it could possibly cause leaky state (if you forget to mount a new one on the  beginning of the test).

Reply:
Rewrote it a bit to avoid this.